packages/@aws-cdk/aws-redshift/lib/parameter-group.ts

```diff
@@ -51,7 +102,7 @@ export class ClusterParameterGroup extends ClusterParameterGroupBase {
    * Imports a parameter group
    */
   public static fromClusterParameterGroupName(scope: Construct, id: string, clusterParameterGroupName: string): IClusterParameterGroup {
-    class Import extends Resource implements IClusterParameterGroup {
+    class Import extends ClusterParameterGroupBase {
```

Comment:
Why this change? This is another indication that the wrong level of abstraction is being used. Does `ClusterParameterGroupBase` extend `Resource`? If not, this change alters fundamentally what this function returns.

Reply:
`ClusterParameterGroupBase` does indeed extend `Resource` and thought that this was a mistake considering the `Import` should implement the abstract methods

Reply:
The import may implement the abstract methods, but I don't think we need to change this at this time; this is out-of-scope.